core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityCheckerMetadata.scala

```diff
@@ -313,10 +316,22 @@ case class Correlations
     corrMeta.putString(SanityCheckerNames.CorrelationType, corrType.sparkName)
     corrMeta.build()
   }
+
+  private[op] def +(corr: Correlations): Correlations =
+    new Correlations(featuresIn ++ corr.featuresIn, values ++ corr.values, nanCorrs ++ corr.nanCorrs, corrType)
```

Comment:
Would the Correlations have to have the same correlation type for this to make sense?

Reply:
yes- but I am not sure it makes sense to error out on that... should I maybe log a warning?

Reply:
added custom to reflect if have multiple corr types